deploy/containers/credhub_setup/auth_test.go

```diff
@@ -0,0 +1,209 @@
+package main
```

Comment:
This is not a unit testing file, therefore, it should be on a separate package and labelled accordingly using Bazel, so we can ignore it while doing only unit testing.